XboxKeyboardMouse/TranslateKeyboard.cs

```diff
@@ -21,24 +21,25 @@ class TranslateKeyboard
         private static readonly Dictionary<Key, X360Buttons> buttonsmap = new Dictionary<Key, X360Buttons>
             {
                 { Key.Space, X360Buttons.A },
-                { Key.F, X360Buttons.B},
-                { Key.E, X360Buttons.X },
-                { Key.Q, X360Buttons.Y},
+                { Key.LeftCtrl, X360Buttons.B},
+                { Key.R, X360Buttons.X },
```

Comment:
i'm not against these changing (this is why we need custom config) but every game i've play has 'X' as 'E'